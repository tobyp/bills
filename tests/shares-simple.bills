#!./bills
A	B.	1.00
A	C.	2.00
B	C.	4.00
