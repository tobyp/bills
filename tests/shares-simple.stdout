todo: C pays 3.0 to A
todo: C pays 3.0 to B
